RELEASES.md

```diff
@@ -0,0 +1,112 @@
+# Version 0.16.0 (2025-08-04)
+
+This release introduces initial support for the **LoongArch CPU architecture**, a major milestone for the project. Version 0.16.0 also significantly expands our Linux ABI compatibility with the addition of **nine new system calls** such as `memfd_create` and `pidfd_open`.
+
+Key enhancements include expanded functionality for **netlink (for uevent notifications)** and **UNIX sockets (for file descriptor passing)**, the first implementation of **CgroupFS**, and a major testing improvement with the integration of system call tests from the **Linux Test Project (LTP)**. We've also adopted **[Nix](https://nix.dev/manual/nix/2.28/introduction)** for building the initramfs, streamlining our cross-compilation and testing workflow.
```

Comment:
> **netlink (for uevent notifications)** and **UNIX sockets (for file descriptor passing)**

It's easy to phrase this sentence as "we added support for netlink/UNIX sockets in version 0.16.0". It isn't true.

 - I don't think we should mention uevent here because the support is incomplete (we can receive no uevent now.... the socket exists but the kernel won't inject any messages into it).
 - If you mention passing file descriptors, then it is equally important to mention that we are adding a new socket type, as I mentioned above.
 - https://github.com/asterinas/asterinas/issues/1963 is not mentioned at all. Do you want to mention it?